src/main/webapp/js/instructorStudentRecords.es6

```diff
@@ -122,6 +126,8 @@ function checkComment(form) {
     }
 }
 
+/* eslint-enable consistent-return */
```

Comment:
You don't need to do this. Just replace `return true;` with `return;` and `return false;` with `e.preventDefault()`, ESLint should not complain anymore.